tools/common/Application.cs

```diff
@@ -697,6 +697,33 @@ public static void UpdateDirectory (string source, string target)
 			FileCopier.UpdateDirectory (source, target);
 		}
 
+		// Duplicate xcode's `builtin-copy` exclusions
+		public static void ExcludeNonEssentialFrameworkFiles (string framework)
+		{
+			// builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -exclude Headers -exclude PrivateHeaders -exclude Modules -exclude \*.tbd
+			File.Delete (Path.Combine (framework, ".DS_Store"));
+			DeleteDir (Path.Combine (framework, "CVS"));
+			DeleteDir (Path.Combine (framework, ".svn"));
+			DeleteDir (Path.Combine (framework, ".git"));
+			DeleteDir (Path.Combine (framework, ".hg"));
+			DeleteDir (Path.Combine (framework, "Headers"));
+			DeleteDir (Path.Combine (framework, "PrivateHeaders"));
+			DeleteDir (Path.Combine (framework, "Modules"));
```

Comment:
Question, what about having a list with the paths we are not interested and loop over it? 
```suggestion
			foreach (var dir in new string [] {"CVS", ".svn", ".git", ".hg", "Headers", "PrivateHeaders", "Modules"}) {
						DeleteDir (Path.Combine (framework, dir));
			}
```
We can even put that in a var and do a unit test.

Reply:
a loop makes sense :)
not sure a unit test is needed - unless it become something more general (not private)
the end goal is to have add the test repo to sample tester and check the tree/file structure